src/Pim/Bundle/EnrichBundle/Normalizer/ProductViolationNormalizer.php

```diff
@@ -40,9 +41,17 @@ public function normalize($violation, $format = null, array $context = [])
             }
 
             $attribute = explode('-', $matches['attribute']);
+            $attributeCode = $attribute[0];
+
+            // TODO: TIP-722 - to revert once the identifier product value is dropped.
+            if ($attributeCode === $this->attributeRepository->getIdentifierCode() &&
+                $violation->getConstraint() instanceof UniqueValue
```

Comment:
No, the interface is `ConstraintInterface`. That would make it ignore all constraints, that is not what I want. I want to ignore only this particular constraint.